apps/ios/Shared/Views/Chat/ChatView.swift

```diff
@@ -221,75 +221,79 @@ struct ChatView: View {
                 }
             }
             ToolbarItem(placement: .navigationBarTrailing) {
-                if selectedChatItems != nil {
-                    Button {
-                        withAnimation {
-                            selectedChatItems = nil
-                        }
-                    } label: {
-                        Text("Cancel")
-                    }
+                if im.isLoading {
+                    ProgressView().modifier(AppearAfter(seconds: 0.5))
```

Comment:
Large diff due to indentation - only added `ProgressView()`

Reply:
indentation is not diff, GitHub hides it